.bit/responses/1.1-Week 1 Step 1.md

```diff
@@ -48,10 +48,10 @@ For our camp, each week is placed inside an issue. Only when you complete the we
 If you want to contribute to someone else's code, you would "fork" it. This creates a copy of the code under your account that you can make changes to. Create a fork when you **want to make changes to someone else's code and contribute to it.**
 
 #### ‼️ Branch
-Creating a **branch** on a repository is like forking a repository. You would do this when you **want to make changes to your code without harming a working version.**
+Creating a new **branch** on a repository is like create another line of development that is isolated from other code. You would do this when you **want to make changes to your code without effecting a working version.**
 
 #### ‼️ Pull Request
-Once you make changes on **a forked repository or another branch,** you might need to bring the changes into the "main" repository. This allows YOUR changes to be visible in the main project! *You are basically asking for permission to "merge" your changes."
+Once you make changes on **a forked repository or another branch,** you might need to bring the changes into the `main` repository. This allows YOUR changes to be visible in the main project! *You are basically asking for permission to "merge" your changes.*
```

Comment:
Proposal for wording: 
"Once you make changes on **a fork or a branch**, you might want to bring the changes into the *main* branch of the branched/forked repository. ..."